### ISSUE TITLE
WowUp and Github compatibility

### ISSUE THREAD

**Issue by scrazzy** [ynazar1/Arh#1]
Hi, perfect job by reviving this great addon! Thank you all who contributed.
Just idea, because this version of addon isnt available on Curse nor Wowinterface, what about making this Github release compatible with WowUp addon manager?
It shouldn't be difficult, according to their documentation, maybe just doing tagged release :)
https://wowup.io/guide/get-addons/overview

Anyway, thanks a lot for your work on Arh.

**Comment by ynazar1** [comment]
@scrazzy give it a try again. I've added the WoWUp release method and quickly checked via the addon (i don't normally use it) and wowup made it show up.

As far as uploading it to other sites, I've made a bit of a point about it in Curse but I don't feel 100% morally correct to re-uploading this to other sites under my name without permission or policy change around abandoned addons. I've tried reaching out to the original authors but so far it's been crickets as the person not been active.
